theme/extensions/tab-center-reborn.css

```diff
@@ -117,3 +118,19 @@
         }
     }
 }
+
+@supports -moz-bool-pref("gnomeTheme.extensions.tabCenterReborn-always-open") {
```

Comment:
Let's call this `gnomeTheme.extensions.tabCenterReborn.alwaysOpen`.

Reply:
No problem will fix his problem and change the name when i have time (i'm with friend today and tomorrow).

Reply:
done with : https://github.com/rafaelmardojai/firefox-gnome-theme/pull/588/commits/a4428fc954eb2b7531f1f5b27a796f6117d833e7